rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -1211,6 +1211,36 @@ void save_saveLastRetiredFederationP2SHScript_afterRSKIP186_getsValueFromStorage
 
     }
 
+    @Test
+    void save_saveFederationElection_shouldSaveInStorage() {
```

Comment:
remove the `save_` prefix

Reply:
`save` is the method name we are testing. We might leave the `save_` prefix and remove it from what comes after, thus making clear what method we are testing. E.g.:

`save_FederationElection_shouldSaveInStorage() `
WDYT guys? @julia-zack @jeremy-then

Reply:
same as [here](https://github.com/rsksmart/rskj/pull/2559#discussion_r1672232989)

Reply:
Updated.